### ISSUE TITLE
<color> Plaid Skirts available on Taxi, but not on Box

### ISSUE THREAD

**Issue by Icychocobo** [d3athrow/vgstation13#9838]
For whatever reason, the Suit Vendors in Box station don't have the various colored Plaid Skirt suits. Taxi Station had them though. Having tested this on a server I hosted myself, even spawned Suit-Vends lack them. No idea why.
Would be neat to seem them added in (or back?)

**Comment by ghost** [comment]
What is the tag about ?

**Comment by Probe1** [comment]
That's an intriguing issue.

**Comment by Probe1** [comment]
Can't seem to find it.  Maybe it was removed at an earlier point in time.

![dreamseeker_2016-05-02_04-51-10](https://cloud.githubusercontent.com/assets/8550416/14950664/95129248-1021-11e6-957d-eb95c00f14c1.png)

**Comment by clusterfack** [comment]
Depending on what name they are given when defined in the code it could be my vending machine change bundling them together or something

**Comment by Icychocobo** [comment]
"What is the tag about ?"
That was actually meant to be a placeholder, not a tag. There's three colored "Plaid Skirt"s, then just a plain "skirt". I wasn't sure how exactly to format the title.

I seem nuts, but I'm absolutely certain that I'm not. I know for sure that they were there a couple nights ago. Around Thursday or Friday I think.

Ahh, you came in just as I was typing, Cluster. That may have been it. I believe it was in the normal suits section, and their names were [color here]plaidskirt.

**Comment by Probe1** [comment]
Now I'm even more intrigued.

Look at this page.  https://github.com/d3athrow/vgstation13/commits/Bleeding-Edge/code/game/machinery/vending.dm

![image](https://cloud.githubusercontent.com/assets/8550416/14951676/bfa492a2-1028-11e6-8df9-a9b5732c472d.png)

If you click the < > it will open up the repository from that moment in time.  I went through over half a dozen and ctrl-f'd "skirt".  The only skirt listing in any of the versions was for the autodrobe.

So in short, I can't find a point between now and last December that they ever had them.

**Comment by Icychocobo** [comment]
How the hell?
Well, it looks like I'm a madman then. However, I did a smidge of digging on my own and found the exact item, which is /obj/item/clothing/under/plaid_skirt/blue. At least, that's what it is on the /tg/station wiki. I can see in there that Plaid, Skirt, Blue, whatever you pick won't find it in coding. So that much is certain.
I know for a fact that I had acquired it through some method. I don't recall if it was roundstart, so the possibility of Bus is real. And I really want to say that I got it from the Vendor.
But, the code doesn't lie. It's not there. It hasn't been there. And that's pretty much it. So I must have just found it.

So, uh... Could we have them added in then? It's pretty frivolous, but may as well make some use out of this issue, since I bothered to waste your time/make the issue.
Oh, and thanks for coming in and digging. It's appreciated.

**Comment by Probe1** [comment]
It's not wasting time when its an interesting mystery.

**Comment by Probe1** [comment]
@Icychocobo you should take a look at the PRs changes https://github.com/d3athrow/vgstation13/pull/9840/files .  This was a very simple request and something you could learn from and do yourself in the future.  It's a nice thing to be able to do and I envy the coders that are far better than me.
